README.md

```diff
@@ -371,30 +370,74 @@ Some common configuration settings can be provided via the command-line:
 ruff path/to/code/ --select F401 --select F403
 ```
 
-See `ruff --help` for more:
+See `ruff check --help` for more:
 
 <!-- Begin auto-generated cli help. -->
 ```
-Ruff: An extremely fast Python linter.
+Run Ruff on the given files or directories (default)
 
-Usage: ruff [OPTIONS] <COMMAND>
+Usage: check [OPTIONS] [FILES]...
```

Comment:
Would it be possible to use `ruff check` instead of `check` only?
```suggestion
Usage: ruff check [OPTIONS] [FILES]...
```

Reply:
I really can't get Clap to do this through the programmatic API. I don't understand why. If I run `cargo run -- help check` or any equivalent command, it does the right thing (includes "ruff check" here).

Reply:
I might have to call the binary directly or something.

Reply:
Have you tried `Args::parse_from(["help", "check"])`?

Reply:
Ah nvm ... it's about the name shown after the `Usage:` .... ok yeah I don't know.

Reply:
I got this working by invoking the binary, same as in the integration tests (maybe there's a cleaner way to do that? Just going with what I know). It's obviously worse than using `Clap` directly, but since this is a dev-only command and will fail in CI if broken, I'm fine with it. The output is better.

Reply:
Hmm, this actually doesn't trigger a build of the `ruff` binary unfortunately.

Reply:
How about just generating the help via the library and doing a `.replace("Usage:", "Usage: ruff")` ... it's a hack but it works.

Reply:
Yeah, that sounds reasonable.

Reply:
Maybe we should report it upstream to improve the API.

Reply:
Reported to clap as https://github.com/clap-rs/clap/issues/4685 :)